style-guide.md

```diff
@@ -0,0 +1,111 @@
+
+## Inclusion
+See also: https://developers.google.com/style/inclusive-documentation
+
+* Use gender-neutral language wherever possible (firefighter vs fireman)
+* Use gender-neutral pronouns when referring to a hypothetical person ("they" instead of "he or she"), even if that requires changing a verb conjugation ("they **adjust** their settings" vs "he or she **adjusts** his or her settings")
+* Avoid terms like master/slave or blacklist/whitelist - use parent/child or denylist/allowlist
+	* blacklist/whitelist can be used as verbs or nouns, while denylist/allowlist tend to only be used as nouns; consider "add to the allowlist" as a replacement for the verb form of "whitelist"
+* Avoid calling tasks "simple" or "easy" as it may frustrate users who are finding it to not be all that easy
+	* It is acceptable to use comparisons, e.g. to say that something is easier than something else
+
+## Logos
+* Before using a new logo for an external organization or moving an existing logo to a place where it has a next context, make a good-faith effort to reach out to the organization for approval if usage of that logo is not clear from their policy
+* Do not use NHLBI logos, including the BioData Catalyst logo, except where it has been explicitly approved
+* Review the logo policy for specific organizations on the Social Media and Outreach Policy document (ask for Google Drive access if necessary)
+
+## Technical detail
+* Generally speaking, assume that your audience is computer literate but may not know their way around the command line, git, Docker, or workflow languages
+	* This will vary - a user seeking out best practices for a specific workflow language will know what a workflow language is
+* Keep track of technical phrases and add them to the glossary if they are not already defined in the glossary
+	* What is considered a "technical phrase" can be nebulous; use your best judgment
+* Focus more on the how-to-do-things than what's happening under the hood
+
+## Timeliness
+* Follow these guidelines: https://developers.google.com/style/timeless-documentation
+* In places where it is truly necessary to anchor something to a point in time, consider adding an RST comment reading `.. !time` nearby so that later maintainers of the documentation will be able to search all such instances and update them as necessary.
+
+## Tone
+Adapted from: https://developers.google.com/style/tone
+* Humor in the form of slightly silly examples is okay, but generally keep things formal
+	* "My Cool Workflow", etc is okay
+* Never include any humor at the expense of a person, organization, or group of people
+* You can make the case for a feature's existence, but avoid sounding too much like a salesperson
+* When telling someone to preform a task in a tutorial, use second-person implicit whenever it makes sense to do so -- e.g., write "Click this button to delete System32" instead of "You can click this button to delete System32"
```

Comment:
it's been a while, what is the difference between implicit and explicit and is it worth picking one or the other?
At least some examples, possibly links to explain to start

Reply:
This implicit/explicit thing caused a lot of confusion, so the current version is more specific and links to an explanation.